specs/MultiProfile.md

```diff
@@ -328,6 +362,13 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   /// See ICoreWebView2CookieManager.
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
+
+[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
+interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
+  /// Delete the profile completely. All webviews on this profile will be closed and  
```

Comment:
We need to document what happens when a CoreWebView2 is created in a profile that's being deleted (like you described in the other comment)